### PR TITLE
azurerm_network_watcher_flow_log  - support for the`interval_in_minutes` property

### PR DESCRIPTION
**Pull request by magodo**
This PR fix issue #5830 

Additionally, fix one bug of example in document, which lacks of specifying `account_kind` to `StorageV2` (which defaults to v1) for storage account, which is required by azurerm_log_analytics_workspace.

### CODE REVIEW

**Comment by magodo** [comment]
image

File: website/docs/r/network_watcher_flow_log.html.markdown [hunk 1]
```diff
@@ -105,6 +107,7 @@ The following arguments are supported:
 * `workspace_id` - (Required) The resource guid of the attached workspace.
 * `workspace_region` - (Required) The location of the attached workspace.
 * `workspace_resource_id` - (Required) The resource ID of the attached workspace.
+* `interval` - (Optional) The interval in minutes which would decide how frequently service should do flow analytics.
```
Comment (katbyte) [inline website/docs/r/network_watcher_flow_log.html.markdown]:
```suggestion
* `interval` - (Optional) How frequently service should do flow analytics in minutes.
```

File: azurerm/internal/services/network/resource_arm_network_watcher_flow_log.go [hunk 2]
```diff
@@ -141,6 +142,13 @@ func resourceArmNetworkWatcherFlowLog() *schema.Resource {
 							Required:     true,
 							ValidateFunc: azure.ValidateResourceIDOrEmpty,
 						},
+
+						"interval": {
```
Comment (katbyte) [inline azurerm/internal/services/network/resource_arm_network_watcher_flow_log.go]: COuld we make this 
```suggestion
						"interval_in_ minutes": {
```

File: azurerm/internal/services/network/resource_arm_network_watcher_flow_log.go [hunk 3]
```diff
@@ -141,6 +142,13 @@ func resourceArmNetworkWatcherFlowLog() *schema.Resource {
...
+							Type:         schema.TypeInt,
+							Optional:     true,
+							ValidateFunc: validation.IntAtMost(math.MaxInt32),
```
Comment (katbyte) [inline azurerm/internal/services/network/resource_arm_network_watcher_flow_log.go]: are negative values allowed?

File: azurerm/internal/services/network/tests/resource_arm_network_watcher_flow_log_test.go [hunk 4]
```diff
@@ -488,6 +511,41 @@ resource "azurerm_network_watcher_flow_log" "test" {
...
+    enabled = true
+    days    = 7
+  }
+
+  traffic_analytics {
+    enabled               = true
+    workspace_id          = "${azurerm_log_analytics_workspace.test.workspace_id}"
+    workspace_region      = "${azurerm_log_analytics_workspace.test.location}"
+    workspace_resource_id = "${azurerm_log_analytics_workspace.test.id}"
+    interval              = 10
```
Comment (katbyte) [inline azurerm/internal/services/network/tests/resource_arm_network_watcher_flow_log_test.go]: This test will fail as this should be
```suggestion
    interval_in_minutes              = 10
```

File: azurerm/internal/services/network/tests/resource_arm_network_watcher_flow_log_test.go [hunk 5]
```diff
@@ -242,6 +243,28 @@ func testAccAzureRMNetworkWatcherFlowLog_trafficAnalytics(t *testing.T) {
...
+					resource.TestCheckResourceAttrSet(data.ResourceName, "resource_group_name"),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "network_security_group_id"),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "storage_account_id"),
+					resource.TestCheckResourceAttr(data.ResourceName, "retention_policy.#", "1"),
+					resource.TestCheckResourceAttr(data.ResourceName, "retention_policy.0.enabled", "true"),
+					resource.TestCheckResourceAttr(data.ResourceName, "retention_policy.0.days", "7"),
+					resource.TestCheckResourceAttr(data.ResourceName, "enabled", "true"),
+					resource.TestCheckResourceAttr(data.ResourceName, "traffic_analytics.#", "1"),
+					resource.TestCheckResourceAttr(data.ResourceName, "traffic_analytics.0.enabled", "true"),
+					resource.TestCheckResourceAttr(data.ResourceName, "traffic_analytics.0.interval", "10"),
```
Comment (katbyte) [inline azurerm/internal/services/network/tests/resource_arm_network_watcher_flow_log_test.go]: THis check will fail?

**Pull request merged**